lidarrmetadata/provider.py

```diff
@@ -936,27 +748,39 @@ def __init__(self):
         self._client = mediawikiapi.MediaWikiAPI()
 
     def get_artist_overview(self, url):
-        return self.get_summary(url)
+        if 'wikidata' in url:
+            title = self.get_wikipedia_title(url)
+        else:
+            title = self.title_from_url(url)
+        return self.get_summary(title)
+
+    def get_wikipedia_title(self, url):
+        entity = self.entity_from_url(url)
+        wikidata_url = 'https://www.wikidata.org/w/api.php?action=wbgetentities&ids=' + entity + '&props=sitelinks&sitefilter=enwiki&format=json'
+        data = requests.get(wikidata_url).json()
+        sitelinks = data['entities'][entity]['sitelinks']
+        if 'enwiki' in sitelinks:
+            return sitelinks['enwiki']['title']
+        return ''
 
     @util.CACHE.memoize()
-    def get_summary(self, url):
+    def get_summary(self, title):
         """
         Gets summary of a wikipedia page
         :param url: URL of wikipedia page
         :return: Summary String
         """
         try:
-            title = self.title_from_url(url)
             return self._client.summary(title, auto_suggest=False)
         # FIXME Both of these may be recoverable
         except mediawikiapi.PageError as error:
-            logger.error('Wikipedia PageError from {url}: {e}'.format(e=error.message, url=url))
+            logger.error(u'Wikipedia PageError from {title}: {e}' % {'e':error, 'title':title})
```

Comment:
I could only get this to work correctly if I used the old-fashioned `%` instead of `.format`.  I have no idea why.

Reply:
It'll probably be fine for now. I'm planning to update everything for py3 when I get a chance and will likely change these to f-strings.